c/tskit/tables.c

```diff
@@ -7818,6 +7850,10 @@ ancestor_mapper_run(ancestor_mapper_t *self)
                     goto out;
                 }
                 current_parent = parent;
+                if (node_time[current_parent] > self->oldest_node_time) {
+                    goto out;
```

Comment:
We only use ``goto out`` for error conditions. You can ``break`` here instead, right?

Reply:
I don't think we can use break, because we have more than one loop to break out of. We end up getting other errors:
```
stderr:
Bug detected in ../tskit/tables.c at line 7802. If you are using tskit directly please open an issue on GitHub, ideally with a reproducible example. (https://github.com/tskit-dev/tskit/issues) If you are using software that uses tskit, please report an issue to that software's issue tracker, at least initially.
```

Reply:
ok -- I put that final call inside another if statement